app/models/user.rb

```diff
@@ -16,6 +16,10 @@ class User < ApplicationRecord
   def recent_posts()
     Post.where(author_id: id).order(created_at: :desc).limit(3)
   end
+
+  def admin?
+    role == 'admin'
+  end
```

Comment:
@SergSkol and @kidde60 this is impressive adding the admin in your user model, keep up the good work